lib/blpapi-wrapper.ts

```diff
@@ -395,7 +427,7 @@ export class Session extends events.EventEmitter {
                     result.options = s.options;
                 }
                 return result;
-            }));
+            }), identity);
```

Comment:
For clarity, it may be best to capture the `_.map` for subscribe in a variable so that the call to `subscribe` appears straightforward.